NuGet.config

```diff
@@ -1,6 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <add key="dotnetcore" value="https://dotnet.myget.org/F/dotnet-core/api/v3/index.json" />
```

Comment:
Do we know what we need to do to avoid referencing this feed? Should MirrorClr pull in more packages?